src/workspaces/features/CreateConnectionDialog/CreateConnectionDialog.tsx

```diff
@@ -110,7 +110,8 @@ export default function CreateConnectionDialog({
             fields:
               connectionType !== ConnectionType.Custom
                 ? convertFieldsToInput(connection, rest)
-                : rest.fields.map((field: FieldForm) => ({
+                : rest.fields &&
```

Comment:
This is not correct since the value of `fields` is going to be "false" instead of an empty list.

`rest.fields?.map() ?? []`is better

Reply:
Fixed, thx